lms/templates/ccx/schedule.html

```diff
@@ -72,26 +73,28 @@
     <h2>${_('Schedule a Unit')}</h2>
     <form role="form" id="add-unit" name="add-unit" class="ccx-form">
       <div class="field">
-        <b>${_('Section')}</b><br/>
-        <select name="chapter"></select>
+        <label for="ccx_chapter"><b>${_('Section')}</b></label>
```

Comment:
Please provide a classname for this label. Something like `form-label` will work, but make sure the styles look good (if applicable).